classifier.py

```diff
@@ -1523,6 +1533,9 @@ def audience(cls, identifier, name):
         if isinstance(identifier, basestring) and identifier.startswith('Y'):
             return cls.AUDIENCE_YOUNG_ADULT
 
+        if isinstance(identifier, basestring) and identifier=='FIC':
+            # FIC is used for all types of fiction.
+            return None
```

Comment:
Previously FIC was treated as an indicator of adult fiction, which is not how FIC is used in the real world.